ArcGISRuntimeSDKQt_CppSamples/Scenes/FeatureLayerExtrusion/FeatureLayerExtrusion.cpp

```diff
@@ -0,0 +1,128 @@
+// [WriteFile Name=FeatureLayerExtrusion, Category=Scenes]
+// [Legal]
+// Copyright 2017 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "FeatureLayerExtrusion.h"
+
+#include "FeatureLayer.h"
+#include "ServiceFeatureTable.h"
+#include "ArcGISTiledElevationSource.h"
+#include "Scene.h"
+#include "SceneQuickView.h"
+#include "OrbitLocationCameraController.h"
+#include "Point.h"
+#include "SimpleLineSymbol.h"
+#include "SimpleFillSymbol.h"
+#include "SimpleRenderer.h"
+#include "RendererSceneProperties.h"
+
+
+using namespace Esri::ArcGISRuntime;
+
+bool showTotalPopulation = true;
+double distance = 20000000;
+double heading = 0;
+double pitch = 55;
+double roll = 0;
+
+FeatureLayerExtrusion::FeatureLayerExtrusion(QQuickItem* parent /* = nullptr */):
+    QQuickItem(parent)
+{
+}
+
+void FeatureLayerExtrusion::init()
+{
+    // Register classes for QML
+    qmlRegisterType<SceneQuickView>("Esri.Samples", 1, 0, "SceneView");
+    qmlRegisterType<FeatureLayerExtrusion>("Esri.Samples", 1, 0, "FeatureLayerExtrusionSample");
+}
+
+void FeatureLayerExtrusion::componentComplete()
+{
+    QQuickItem::componentComplete();
+
+
+
+    // Create the feature service to use
+    m_featureTable = new ServiceFeatureTable(QUrl("http://sampleserver6.arcgisonline.com/arcgis/rest/services/Census/MapServer/3"),this);
+
+    // load all fields in the service feature table
+    QueryParameters queryParams = QueryParameters();
```

Comment:
this line can be changed to `QueryParameters queryParams;`

Reply:
I was instructed by Trevor to remove the query stuff. 
" So we can drop all lines looking like:
// load all fields in the service feature table
    QueryParameters queryParams = new QueryParameters();
    queryParams.setWhereClause("1=1");
    statesServiceFeatureTable.queryFeaturesAsync(queryParams, ServiceFeatureTable.QueryFeatureFields.LOAD_ALL);"